apps/desktop/e2e/tests/drag-file.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { spawnAndLog } from '../utils.js';
+import { codeForSelectors as dragAndDrop } from 'html-dnd';
+
+const TEMP_DIR = '/tmp/gitbutler-drag-files';
+const REPO_NAME = 'simple-drag-test';
+
+describe('Drag', () => {
+	before(() => {
+		spawnAndLog('bash', [
+			'-c',
+			`
+				source ./e2e/scripts/init.sh ${TEMP_DIR}
+				bash ./e2e/scripts/confirm-analytics.sh
+				cd ${TEMP_DIR};
+				git clone remote ${REPO_NAME} && cd ${REPO_NAME}
+				setGitDefaults
+				$CLI project -s test add --switch-to-workspace "$(git rev-parse --symbolic-full-name "@{u}")"
+				$CLI branch create virtual-one
+				$CLI branch create virtual-two
+				echo "hello world" > hello
+			`
+		]);
+	});
+
+	it('drag file from one lane to another', async () => {
+		const fileSelector = '[data-testid="file-hello"]';
+		const dropSelector = '[data-testid="virtual-two-files-dz"]';
+
+		const fileItem = await $(fileSelector);
+		const dropTarget = await $(dropSelector);
+		await fileItem.waitForDisplayed();
+		await dropTarget.waitForDisplayed();
+
+		// The actual drop target can be different from the element with the `dropZone` directive..
+		await driver.executeScript(dragAndDrop, [fileSelector, dropSelector + ' .dropzone-target']);
```

Comment:
@ndom91 this used to work - likely the difference is on our side somewhere. No rush at all, but if you could at any point help me debug this? 🙏 

From last time, I remember that the change in `dropzone.ts` was important. The current problem might be similar or related.

Reply:
Actually, I resolved it in the last force push.